### PR TITLE
[TASK] Test cross-chain withdrawals with xanchor + anchor

### PR DESCRIPTION
**Pull request by drewstone**
Test cross-chain withdrawals with xanchor + anchor according to issue #155

### CODE REVIEW

File: pallets/anchor/src/lib.rs [hunk 1]
```diff
@@ -385,6 +383,10 @@ impl<T: Config<I>, I: 'static> AnchorInterface<AnchorConfigration<T, I>> for Pal
 		bytes.extend_from_slice(&fee_bytes);
 		bytes.extend_from_slice(&refund_bytes);
 		bytes.extend_from_slice(&commitment.encode());
+		println!("WITHDRAW PROOF bytes");
```
Comment (drewstone) [inline pallets/anchor/src/lib.rs:386]: We should clean these up @dharjeezy

File: pallets/xanchor/src/lib.rs [hunk 2]
```diff
@@ -682,17 +682,21 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 				.encode()
 				.into(),
 			};
-			println!("{:?}", update_edge);
+			println!("{:?}", "updating edge");
```
Comment (drewstone) [inline pallets/xanchor/src/lib.rs:685]: These too

**Review by drewstone — COMMENTED** [review]
Great work @dharjeezy. Let's clean this up and we'll prepare to debug with @filiplazovic once the integration tests land.

File: pallets/xanchor/src/tests.rs [hunk 3]
```diff
@@ -234,7 +275,7 @@ fn should_bridge_anchors_using_xcm() {
 	});
 
 	ParaB::execute_with(|| {
-		setup_environment(Curve::Bn254);
+		//setup_environment(Curve::Bn254);
```
Comment (dutterbutter) [inline pallets/xanchor/src/tests.rs:278]: Is commenting this out intentional or can it be removed?

Comment (dharjeezy) [reply pallets/xanchor/src/tests.rs]: I was trying to check something out. I have reverted now.

**Pull request closed without merging**